---
layout: post
title: "Cell-type deconvolution from DNA methylation: a review of recent applications"
date:   2017-7-16
excerpt: "Our recent review discussing the opportunities and applications in cell-type deconvolution."
image: "/images/figure1-cube.png"
categories: publications
tags: dna-methylation biostatistics
---

**Titus AJ**, Gallimore R, Salas LA, Christensen BC (2017) Cell-type deconvolution from DNA methylation: a review of recent applications. *Human Molecular Genetics*. [[PMID: 28977446](https://www.ncbi.nlm.nih.gov/pubmed/?term=28977446)].

**Abstract:** Recent advances in cell-type deconvolution approaches are adding to our understanding of the biology underlying disease development and progression. DNA methylation (DNAm) can be used as a biomarker of cell types, and through deconvolution approaches, to infer underlying cell type proportions. Cell-type deconvolution algorithms have two main categories: reference-based and reference-free. Reference-based algorithms are supervised methods that determine the underlying composition of cell types within a sample by leveraging differentially methylated regions (DMRs) specific to cell type, identified from DNAm measures of purified cell populations. Reference-free algorithms are unsupervised methods for use when cell-type specific DMRs are not available, allowing scientists to estimate putative cellular proportions or control for potential confounding from cell type. Reference-based deconvolution is typically applied to blood samples and has potentiated our understanding of the relation between immune profiles and disease by allowing estimation of immune cell proportions from archival DNA. Bioinformatic analyses using DNAm to infer immune cell proportions, part of a new field known as Immunomethylomics, provides a new direction for consideration in epigenome wide association studies (EWAS).
